index.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+// See https://github.com/jshint/jshint/issues/1747 for context
+/* global -Promise */
+var Promise = require('es6-promise').Promise;
+var adb = require('adbkit');
+var portfinder = require('portfinder');
+
+var REMOTE_DEBUGGER_SOCKET = 'localfilesystem:/data/local/debugger-socket';
+
+module.exports = forwardPorts;
+
+function forwardPorts(options) {
+  options = options || {};
+  var devices = options.devices;
+
+  var client = adb.createClient();
```

Comment:
Good point - I'll submit a PR to find-devices too, since I just named it `client` there, too.